Source/Utils/Parser/ParserUtils.cpp

```diff
@@ -43,7 +43,8 @@ namespace {
         // (2.0*(numeric_limits<int>::max()/2+1)) converts numeric_limits<int>::max()+1 to a real ensuring accuracy to all digits
         // This accepts x = 2**31-1 but rejects 2**31.
         using namespace amrex::literals;
-        constexpr amrex::Real max_range = (2.0_rt*static_cast<amrex::Real>(std::numeric_limits<int_type>::max()/2+1));
+        constexpr int_type half_max_plus_one = std::numeric_limits<int_type>::max()/2+1;
```

Comment:
Just curious - which check did this fail?

Reply:
This one:
[bugprone-integer-division](https://releases.llvm.org/14.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/bugprone-integer-division.html) "Finds cases where integer division in a floating point context is likely to cause unintended loss of precision."
(in this case it is a false positive, but I changed the code to silence the warning because I think that this check can be quite useful)